src/Uno.UI.RemoteControl/HotReload/HotReloadStatusView.xaml

```diff
@@ -85,6 +85,13 @@
 		</controls:Grid>
 	</DataTemplate>
 
+	<Style x:Key="IndicatorButtonStyle" TargetType="Button">
```

Comment:
@dr1rrb shouldn't we override the whole style of the visual to fluent style to avoid interactions from the app's styles?

Reply:
Yes. @rajamatt you should put this style as implicit style in the `DiagnosticOverlay.Ressources`, so it will impact all buttons that are put in the overlay

Reply:
The button is able to find the style from DiagnosticsOverlay Resources but only if its set explicitly, is this because the Button is in a template?

I just pushed the changes, not sure if this is the right place to use MergedDictionaries either.

Reply:
It's best to reimport the whole winui styles at the root of the overlay, like this: https://github.com/unoplatform/uno.templates/blob/b05ffa1dcbba8e890ad33c0421c8171573ffc6ee/src/Uno.Templates/content/unoapp/MyExtensionsApp.1/App.xaml#L14.